chia/protocols/shared_protocol.py

```diff
@@ -28,3 +29,6 @@ class Handshake(Streamable):
     server_port: uint16
     node_type: uint8
     capabilities: List[Tuple[uint16, str]]
+
+
+capabilities = [(uint16(Capability.BASE.value), "1"), (uint16(Capability.BLOCK_HEADERS.value), "2")]
```

Comment:
The intent here was that "1" would be enabled, so each capability would have "1" as the value. But we have not standardized or documented any of this, this is the first use of capabilities.

Reply:
ok, added some docs and made it so that "1" is considered when checking what features are enabled